src/node.cc

```diff
@@ -3040,6 +3047,20 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--v8-options") == 0) {
       new_v8_argv[new_v8_argc] = "--help";
       new_v8_argc += 1;
+    } else if (strncmp(arg, "--cipher-list=", 14) == 0) {
```

Comment:
Could this be done in `src/node.js` instead of in `src/node.cc`? It seems it would just be less error prone and shorter.